packages/contracts-bedrock/src/L1/OptimismPortal.sol

```diff
@@ -97,30 +96,29 @@ contract OptimismPortal is Initializable, ResourceMetering, ISemver {
 
     /// @notice Reverts when paused.
     modifier whenNotPaused() {
-        require(paused == false, "OptimismPortal: paused");
+        require(paused() == false, "OptimismPortal: paused");
         _;
     }
 
     /// @notice Semantic version.
-    /// @custom:semver 1.11.0
-    string public constant version = "1.11.0";
+    /// @custom:semver 2.0.0
+    string public constant version = "2.0.0";
 
     /// @notice Constructs the OptimismPortal contract.
     /// @param _l2Oracle Address of the L2OutputOracle contract.
-    /// @param _guardian Address that can pause withdrawals.
-    /// @param _paused Sets the contract's pausability state.
     /// @param _systemConfig Address of the SystemConfig contract.
-    constructor(L2OutputOracle _l2Oracle, address _guardian, bool _paused, SystemConfig _systemConfig) {
+    /// @param _superchainConfig Address of the SuperchainConfig contract.
+    constructor(L2OutputOracle _l2Oracle, SystemConfig _systemConfig, SuperchainConfig _superchainConfig) {
         L2_ORACLE = _l2Oracle;
-        GUARDIAN = _guardian;
         SYSTEM_CONFIG = _systemConfig;
-        initialize(_paused);
+        initialize(_superchainConfig);
```

Comment:
We should initialize to `address(0)` here - no need to pass the superchain config to the constructor